### ISSUE TITLE
can you help me in dlib training

### ISSUE THREAD

**Issue by ghost** [LI-ZONG-HAN/Face_Recognition#1]
hi, I see your comment at dlib Issue, I found you used dlib face recognition before, when I used dnn_metric_learning_on_images_ex.cpp to train, I found the result on training datasets is good, but on test datasets is really bad.
I just changed load_mini_batch(33,13), and steps without apparent progresses(10000), the datasets I used has 5M images and 80K ids.
can you ask how you training your face recognition model in dlib?
thanks

**Comment by LI-ZONG-HAN** [comment]
I just used the dlib face recognition model but did not use dlib to train a new model. I trained my own model with Tensorflow. What is the training batch size for the bad result model, (5,5) or (33,13)? In my experience, big batch size is very importance at training big models and big datasets. If batch size is not big enough such as (5,5), training is hard to converge.

**Comment by ghost** [comment]
thanks for your quickly reply, due to my project is highly required on fast inference time, so I may not choose tensorflow , the batch size I chose in dlib is (33,13), but the result is not as good as my thought, I am trying to change the loss function in dlib's face recognition.
